tests/acceptance/pageObjects/personalPage.js

```diff
@@ -242,6 +242,7 @@ module.exports = {
         .waitForElementVisible('@dialogConfirmBtnEnabled')
         .click('@dialogConfirmBtnEnabled')
         .waitForElementNotPresent('@dialog')
+        .waitForElementVisible('@fileUploadStatus')
         .waitForAjaxCallsToStartAndFinish()
```

Comment:
I don't think there will be any ajax calls after the file upload has finished, or?
Ether put this before `.waitForElementVisible('@fileUploadStatus')` or delete it